README.md

```diff
@@ -39,7 +35,7 @@ $ sudo systemctl reboot
 Verify: SSH to the Atomic Host:
 
 ```
-[tguha@dhcp193-94 ~]$ ssh atomic-user@192.168.121.221
+[tguha@dhcp193-94 ~]$ ssh atomic-user@192.168.122.221
```

Comment:
just keep it simple:
```
$ ssh atomic-user@192.168.122.221 
```

Avoid specifying your output too much. In this case PS1 ;-)